src/main/java/com/elca/jobfairmanagementsystem/controller/CandidateController.java

```diff
@@ -0,0 +1,67 @@
+package com.elca.jobfairmanagementsystem.controller;
+
+import com.elca.jobfairmanagementsystem.dto.CandidateDto;
+import com.elca.jobfairmanagementsystem.exception.CandidateNotFoundException;
+import com.elca.jobfairmanagementsystem.service.CandidateService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * @author ghr
+ */
+@RestController
+@CrossOrigin(origins = "*", maxAge = 3600)
+@RequestMapping("/candidate")
```

Comment:
In REST standards, the path of the rest denotes the entity being represented:

Eg example of Customer and Account

/api/customers
/api/accounts
will be the base APIs

Example CRUD and Actions
GET /api/customers -> find all
GET /api/customers/{id} -> find by id
POST /api/customers -> create customer
PUT /api/customer/{id} -> update customer
PATCH /api/customer/{id}/** -> partially update customer
POST /api/customer/{id}/** -> action on customer where ** is the action
DELETE  /api/customer/{id} -> delete customer
GET /api/customer/self -> retrieve customer related to auth

Resources of resources are shown via the path:

GET /api/customers/{id}/accounts -> find accounts for customer